lib/public_key/src/pubkey_cert.erl

```diff
@@ -280,7 +280,7 @@ validate_signature(Cert, DerCert, Key, KeyParams0,
 %% Description: Extracts data from DerCert needed to call public_key:verify/4.
 %%--------------------------------------------------------------------
 verify_data(DerCert) ->
-    {ok, OtpCert} = pubkey_cert_records:decode_cert(DerCert),
+    {ok, OtpCert} = pubkey_cert_records:decode_cert(DerCert, relaxed),
```

Comment:
I think that this must be made configurable from an additional argument for pkix_verify that should be an option list.

Reply:
I have been considering this, but the function later does not extract any issuer- or subject-related information from the certificate. That information can be extracted as long as the certificate can be decoded, empty RDNs or not. With that in mind, do you still want it configurable?